bin/compile/function_metadata/metadata_builder.dart

```diff
@@ -119,6 +123,7 @@ Iterable<CompileTimeFunctionData> getFunctionData(
         name = getAnnotationData(nameAnnotations.first.elementAnnotation!)
             .getField('name')!
             .toStringValue()!;
+        locaizedNames = nameAnnotations.first.arguments?.arguments.last;
```

Comment:
If no localized names are passed, this will result in the normal name being selected.
I.e `@Name('foo')` will result in both `name` and `localizedNames` being set to `foo`.

Reply:
What's the point of doing this? If it's null, first argument will be taken as fallback.
That would be redundant too i think.

Reply:
It's an issue because if `localizedNames` ends up being the non-localized name expression, the generated file will attempt to put a `String` expression into an argument of type `Map<Locale, String>` in the parameter data.

Reply:
Ohh, okay, my bad, I misread your sentence, thought that you were asking to add this behaviour.